app/policies/documentation_policy.rb

```diff
@@ -11,4 +11,8 @@ def edit?
   def update?
     create?
   end
+
+  def show?
+    create?
```

Comment:
this is probably the wrong policy for this, but i'll make a separate story to fix the policies

Reply:
👍